src/neptune/common/backends/utils.py

```diff
@@ -45,14 +52,42 @@
     NeptuneInvalidApiTokenException,
     NeptuneSSLVerificationError,
     Unauthorized,
+    WritingToArchivedProjectException,
 )
 
+if TYPE_CHECKING:
+    from bravado.requests_client import RequestsResponseAdapter
+
 _logger = logging.getLogger(__name__)
 
 MAX_RETRY_TIME = 30
 retries_timeout = int(os.getenv(NEPTUNE_RETRIES_TIMEOUT_ENV, "60"))
 
 
+def ensure_json_body(response: "RequestsResponseAdapter") -> Dict[str, Any]:
+    try:
+        return response.json() or dict()
+    except Exception:
```

Comment:
I think that we can make this exception more precise

https://github.com/psf/requests/blob/6e5b15d542a4e85945fd72066bb6cecbc3a82191/requests/models.py#L948

Reply:
Sure. Thanks! Done